### PR TITLE
[patch] add go binary version during build time

### PR DESCRIPTION
**Pull request by WindzCUHK**
details: https://github.com/yahoojapan/athenz-client-sidecar/pull/25

### CODE REVIEW

**Comment by WindzCUHK** [comment]
pipeline output:
https://app.circleci.com/jobs/github/yahoojapan/authorization-proxy/607/parallel-runs/0/steps/0-103
https://app.circleci.com/jobs/github/yahoojapan/authorization-proxy/609/parallel-runs/0/steps/0-106

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,8 +53,17 @@ jobs:
       - run:
           name: docker image build
           command: |
-            # env DOCKER_BUILDKIT=1 docker build -t ${REPO_NAME}/${IMAGE_NAME}:latest .
-            docker build -t ${REPO_NAME}/${IMAGE_NAME}:latest .
+            TAG=`cat ./.tag`
+            if [ ! -z "$TAG" ]; then
+              docker build --build-arg APP_VERSION=${TAG} -t ${REPO_NAME}/${IMAGE_NAME}:latest .
+            else
+              # env DOCKER_BUILDKIT=1 docker build -t ${REPO_NAME}/${IMAGE_NAME}:latest .
```
Comment (kyfujisa) [inline .circleci/config.yml]: Forgot to delete comment?

Comment (WindzCUHK) [reply .circleci/config.yml]: exist on the 1st day, just leave it?
https://github.com/yahoojapan/authorization-proxy/commits/master/.circleci/config.yml
https://github.com/yahoojapan/authorization-proxy/commit/bb85478cc9f968b1a4c19ed3363152d9a8bbd358#diff-1d37e48f9ceff6d8030570cd36286a61

`DOCKER_BUILDKIT` will clean out old docker build log in stdout, not good for pipeline

Comment (kyfujisa) [reply .circleci/config.yml]: > exist on the 1st day, just leave it?
It does not matter.

**Pull request merged**